test/Renci.SshNet.Tests/Renci.SshNet.Tests.csproj

```diff
@@ -3,6 +3,10 @@
     <TargetFrameworks>net462;net6.0;net7.0;net8.0</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net462' ">
+    <DefineConstants>$(DefineConstants);NET462_TARGET_FRAMEWORK</DefineConstants>
```

Comment:
If we revert the FIPS test changes, then this is already unnecessary, but FYI the constant `NET462` is defined [by default](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation) so one can do `#if NET462` without any additional configuration

Reply:
OK, removed it.